src/components/layout/top-navigation.tsx

```diff
@@ -43,11 +46,21 @@ const TopNavigation = () => {
   useMatches();
   const pathParams = useParams({ strict: false });
 
+  const findUserQuery = useFindUserQuery({ userId: auth?.token.userId });
+  const listNotificationEventsQuery = useListNotificationEventsQuery({
```

Comment:
Is there a need for an interval for polling the API for new notifications, as we currently don't have an event channel?

It could be done by creating a separate query hook for polling the endpoint for the single latest event for the user, and cache it's notificationEventId, so we can see whether the latest event is the same as the previous one. If it isn't, we could update the hook return value, which could be used to refetch the actual listNotificationEvents query.